package.json

```diff
@@ -5,7 +5,7 @@
     "build-md-tables-from-openapi": "ts-node ./scripts/build-md-tables-from-openapi.ts",
     "readme-fix-reference-docs": "ts-node ./scripts/readme-fix-reference-docs.ts",
     "remove-stoplight-tags-from-openapi": "ts-node ./scripts/remove-stoplight-tags-from-openapi.ts",
-    "manage-project": "ts-node ./scripts/manage-project.ts",
+    "manage-project": "ts-node ./scripts/build-md-tables-from-openapi.ts && ts-node ./scripts/manage-project.ts",
```

Comment:
Dodałem budowanie markdownów, jako hotfix. W ten sposób jeżeli jakiś markdown się nie będzie chciał zbudować to przynajmniej będzie error w konsoli.
Generalnie do poprawy w przyszłym tygodniu, by przy błędzie budowania mardownów, wyrzucało skypt.